src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String MIN_LIMIT_AGE_MESSAGE = "User age is less than min age";
+    private static final String MIN_LENGTH_PASSWORD_MESSAGE
+            = "Password needs 6 characters at least";
+    private static final String NULL_PASSWORD_MESSAGE = "Password is null";
+    private static final String NULL_LOGIN_MESSAGE = "Login is null";
+    private static final String NULL_AGE_MESSAGE = "Age is null";
+    private static RegistrationService registrationService;
+
+    private static StorageDao storageDao;
+    private static User userUnderTest;
+
+    @BeforeAll
+    private static void setup() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    private void beforeEach() {
+        userUnderTest = new User();
+        userUnderTest.setAge(17);
```

Comment:
17 is invalid age. You can set valid data here and change it in method where it is needed